<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Goniometer Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>359</width>
       <height>129</height>
      </rect>
     </property>
     <widget class="QPlainTextEdit" name="lastActions">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>361</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="iniTab">
     <attribute name="title">
      <string>Ini file</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>191</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Transition</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Process:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Mixing Ratio:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Q value: </string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_type">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_process">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_MixingRatio">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_Q">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="suffix">
        <string> keV</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_M">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>171</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Mother</string>
      </property>
      <widget class="QLabel" name="lab_ZM">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_AM">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_RM">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Radius:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_JM">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Spin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Beta2M">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Beta2:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Beta4M">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Beta4:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Beta6M">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>226</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Beta6:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_PiM">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Parity:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_ZM">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Atomic number</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_AM">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Mass number</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_RM">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Charge radius</string>
       </property>
       <property name="suffix">
        <string> fm</string>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_JM">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Spin in units of hbar</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_PiM">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_Beta2M">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>170</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_Beta4M">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_Beta6M">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>File info</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Current file:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Filename placeholder</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Load .ini file</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Save as...</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_M_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>191</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Daughter</string>
      </property>
      <widget class="QLabel" name="lab_ZD">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_AD">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>A:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_RD">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Radius:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_JD">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Spin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Beta2D">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Beta2:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Beta4D">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Beta4:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_Beta6D">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>226</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Beta6:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lab_PiD">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>140</y>
         <width>67</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Parity:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_ZD">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Atomic number</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sb_AD">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>48</width>
         <height>27</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Mass number</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_RD">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>81</width>
         <height>27</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Charge radius</string>
       </property>
       <property name="suffix">
        <string> fm</string>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_JD">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>Spin in units of hbar</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_PiD">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_Beta2D">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_Beta4D">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsb_Beta6D">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>69</width>
         <height>27</height>
        </rect>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="configTab">
     <attribute name="title">
      <string>Config file</string>
     </attribute>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
